pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -534,6 +538,40 @@ func (f *framework) GetWaitingPod(uid types.UID) WaitingPod {
 	return f.waitingPods.get(uid)
 }
 
+// ListPlugins returns a map of extension point name to plugin names configured at each extension
+// point. Returns nil if no plugins where configred.
+func (f *framework) ListPlugins() map[string][]string {
+	m := make(map[string][]string)
+	insertPluginList(&f.queueSortPlugins, m)
+	insertPluginList(&f.preFilterPlugins, m)
+	insertPluginList(&f.filterPlugins, m)
+	insertPluginList(&f.postFilterPlugins, m)
+	insertPluginList(&f.scorePlugins, m)
+	insertPluginList(&f.reservePlugins, m)
+	insertPluginList(&f.permitPlugins, m)
+	insertPluginList(&f.preBindPlugins, m)
+	insertPluginList(&f.bindPlugins, m)
+	insertPluginList(&f.postBindPlugins, m)
+	insertPluginList(&f.unreservePlugins, m)
```

Comment:
nit: I believe we have multiple places calling the same function on different kinds of plugins. It'd be good to add a function to "get all plugins", then:

```go
for _, plugin := range f.getAllPlugins() {
        insertPluginList(&f, m)
}
```

Otherwise, someday if we add a plugin phase, we have to change all the places.

Reply:
sg, but lets do that in a separate PR once [Aldo's PR](https://github.com/kubernetes/kubernetes/pull/83506) is merged.

Reply:
ok, I updated the code to do that, PTAL. I still need to squash.